Sources/Models/SubscriberState.swift

```diff
@@ -55,17 +56,25 @@ public enum SubscriberState {
   public var isNonSubscriber: Bool {
     switch self {
     case .teammate(status: .active, _),
-         .owner(hasSeat: _, status: .active, _):
+         .teammate(status: .trialing, _),
+         .owner(hasSeat: _, status: .active, _),
+         .owner(hasSeat: _, status: .trialing, _):
       return false
```

Comment:
Could also stop matching against `status` here and below and `return self.isActive`, but can do that later.